kyber/__init__.py

```diff
@@ -44,6 +45,8 @@ def deploy_app(tag, force):
         click.echo("Can't find a docker for {}\naborting..".format(app.image))
         return
 
+    click.confirm("About to deploy {} to {}, continue?".format(tag, context.name), abort=True)
```

Comment:
Good suggestion, I was thinking about showing more information, no reason not to do it. Also good point on having an option to disable the prompt for CI, will update that.